lms/djangoapps/instructor/views/api.py

```diff
@@ -1458,20 +1458,9 @@ def get_proctored_exam_results(request, course_id):
     """
     get the proctored exam resultsreport for the particular course.
     """
-    query_features = [
-        'user_email',
-        'exam_name',
-        'attempt_code',
-        'allowed_time_limit_mins',
-        'is_sample_attempt',
-        'started_at',
-        'completed_at',
-        'status',
-    ]
-
```

Comment:
I moved this list of headings into misc.py where the method actually gets called. The new column headings exceed the task limit of 255 characters, and since the list is essentially a constant anyway there's no real reason to keep writing it to the db.